### PR TITLE
RND trees' reassembly

### PR DESCRIPTION
**Pull request by AlexMorgan3817**
<h1>FILE code/game/machinery/autodoc.dm ISN'T INCLUDED IN COMPILE</h1>

### Autodoc need fix

- [x] Force it to heal upon activation

- [x] Make him show the sprite when it appears, and not be invisible

- [x] Remove/Fix autodoc-machinery.

<s>Now it's time to admire my HTML</s>
### Trees
<details>

<summary><div title="Success">SuccessfulTrees Diff</div></summary>

#### BioTech <hr>
Was: bio
Now: image

#### Combat <hr>
Was: combat
Now: image
#### Power <hr>
Was:
power
Now:
image
#### Engineering <hr>
Was:
eng
Now:
image

#### Robotics <hr>
Was:
robo
Now:
image

#### BlueSpace <hr>
Was: image
Now:
image

</details>

## Changelog
:cl:
refactor: RND trees' refactored
add: Autodoc fixed and merged
/:cl:

### CODE REVIEW

File: code/game/machinery/autodoc.dm [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+	if(stat & (NOPOWER|BROKEN))
+		autodoc_processor.set_patient(L)
+		ui_interact(L)
+		update_use_power(2)
+		L.set_machine(src)
+	update_icon()
+
+/obj/machinery/autodoc/affect_grab(var/mob/user, var/mob/target)
+	if (src.occupant)
+		user << SPAN_NOTICE("The autodoc is already occupied!")
```
Comment (ACCount12) [inline code/game/machinery/autodoc.dm]: Use `to_chat`. I'm fairly sure that with goonchat being merged, none of those << messages would even display.

Comment (AlexMorgan3817) [reply code/game/machinery/autodoc.dm]: > Use `to_chat`. I'm fairly sure that with goonchat being merged, none of those << messages would even display.

Isn't my code, ok, i will replace it

**Comment by AlexMorgan3817** [comment]
@ACCount12, changes already committed

File: code/game/machinery/autodoc.dm [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+		return
+	set_occupant(usr)
+	src.add_fingerprint(usr)
+	return
+
+/obj/machinery/autodoc/proc/go_out()
+	if (!occupant || locked)
+		return
+	if(autodoc_processor.active)
+		to_chat(usr, SPAN_WARNING("Autodoc is locked down! Abort all oberations if you need to go out or wait untill all operations would be done."))
```
Comment (drexample) [inline code/game/machinery/autodoc.dm]: until has one "l"

**Pull request merged**